ota-plus-web/app/reactapp/src/partials/OperationCompletedInfo.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
```

Comment:
Please keep up file naming convention throughout the app (`.jsx` for components)